pkg/runs/humanReporter.go

```diff
@@ -178,13 +203,40 @@ func InterrimProgressReportAsString(
 	fmt.Fprintln(&buff, "*** ----------------------------------------------------------------------------")
 	fmt.Fprintf(&buff, "*** run status, ready=%v, submitted=%v, finished=%v, lost=%v\n", ready, submitted, finished, lost)
 	fmt.Fprintf(&buff, "*** throttle=%v\n", throttle)
+
 	if len(resultCounts) > 0 {
-		resultsSoFar := "*** results so far"
-		for result, count := range resultCounts {
-			resultsSoFar = resultsSoFar + fmt.Sprintf(", %v=%v", result, count)
+		resultsSoFar := fmt.Sprintf("*** Results so far:\n*** Total=%v", totalResults)
+
+		orderedKeys := orderResultKeys(resultCounts)
+		for i := range orderedKeys {
+			resultsSoFar = resultsSoFar + fmt.Sprintf(", %v=%v", orderedKeys[i], resultCounts[orderedKeys[i]])
 		}
 		fmt.Fprintln(&buff, resultsSoFar)
 	}
+
 	fmt.Fprintln(&buff, "***")
 	return buff.String()
 }
+
+func orderResultKeys(resultCounts map[string]int) []string {
+
+	var keys []string
+	keys = append(keys, RESULT_PASSED)
+	keys = append(keys, RESULT_PASSED_WITH_DEFECTS)
+	keys = append(keys, RESULT_FAILED)
+	keys = append(keys, RESULT_FAILED_WITH_DEFECTS)
+	keys = append(keys, RESULT_LOST)
+	keys = append(keys, RESULT_ENVFAIL)
+
+	var customLabels []string
+	for keyLabel := range resultCounts {
+		if keyLabel != RESULT_PASSED && keyLabel != RESULT_PASSED_WITH_DEFECTS && keyLabel != RESULT_FAILED && keyLabel != RESULT_FAILED_WITH_DEFECTS && keyLabel != RESULT_ENVFAIL && keyLabel != RESULT_LOST {
```

Comment:
This if statement is brittle. If I added another RESULT key I would need to find and change this line else a bug would occur... 
Better to add all the keys you care about to a map, then this if method becomes "does this key exist in my map"... then if another key is added you don't need to change this code.

Reply:
> It's trivial to check if a specific map key exists by using the value, ok := yourmap[key] idiom. Since you aren't interested in the value, you might also create a map[string]struct{} for example. Using an empty struct{} here has the advantage that it doesn't require any additional space and Go's internal map type is optimized for that kind of values. Therefore, map[string] struct{} is a popular choice for sets in the Go world.

That was from https://stackoverflow.com/questions/10485743/contains-method-for-a-slice